### ISSUE TITLE
A Gentle Introduction to Weight Constraints to Reduce Generalization Error in Deep Learning

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#8785]
A Gentle Introduction to Weight Constraints to Reduce Generalization Error in Deep Learning<br>
Weight regularization methods like weight decay introduce a penalty to the loss function when training a neural network to encourage the network to use small weights.<br>
https://ift.tt/2R647Kp
